### ISSUE TITLE
Add Dotplot View Session Objects to Jekyll Site YML

### ISSUE THREAD

**Issue by ctcncgr** [legumeinfo/LIS-autocontent#28]
https://jbrowse.org/jb2/docs/urlparams/#dotplot-view

**Comment by adf-ncgr** [comment]
@ctcncgr I think we need some further changes here as well:
- in the session spec, tracks should be populated with the paf file so that the user doesn't have to go through any extra work to turn it on (otherwise, they'll just see an empty grid, which is boring)
- currently, it's writing a single Dotplot view for a given genome, even when the genome has multiple comparisons. I think what we want is a link to each comparison. Possibly this would "just work" if you were to update the naming, e.g. instead of `name: JBrowse2 aradu.V14167.gnm2.wga ` use `name: JBrowse2 aradu.V14167.gnm2.x.araca.K10017.gnm1` (or whatever the comparison is). Not sure if the name is getting used as a key, but that's my theory as to why we only get one of these per genome.

let me know if any of that is unclear

**Comment by ctcncgr** [comment]
@adf-ncgr should be good to go. The paf file (without the gz) has been added to tracks and the synteny entries should be added for all comparisons.

**Comment by adf-ncgr** [comment]
Yes, I think it's working. Haven't yet tried the output in the context of an actual jekyll site, but at least the session spec URLs are working so I imagine it will work as expected. I'll leave it open until verified, though.

**Comment by adf-ncgr** [comment]
OK, looks like it is working fine at : https://dev.peanutbase.org/taxa/arachis/
@sdash-github have a look (search for "dotplot" in the page and follow the link). Just let us know if you see any further refinements to the basic idea that would be desirable.
